### ISSUE TITLE
powerline-shell error with 'uptime' segment

### ISSUE THREAD

**Issue by noblehelm** [banga/powerline-shell#291]
Hello,

I'm trying to use 'uptime' as segment but I'm getting an error that goes like this:

```
Traceback (most recent call last):
  File "/usr/bin/powerline-shell", line 11, in <module>
    sys.exit(main())
  File "/home/jr/.local/lib/python3.6/site-packages/powerline_shell/__init__.py", line 186, in main
    segment.add_to_powerline()
  File "/home/jr/.local/lib/python3.6/site-packages/powerline_shell/segments/uptime.py", line 11, in add_to_powerline
    raw_uptime = re.search('(?<=up).+(?=,\s+\d+\s+user)', output).group(0)
  File "/usr/lib/python3.6/re.py", line 182, in search
    return _compile(pattern, flags).search(string)
TypeError: cannot use a string pattern on a bytes-like object
```

**Comment by b-ryan** [comment]
Thanks for the issue - I am going to release the fix for this shortly. It will be fixed in version 0.2.2 -- you will be able to upgrade using `pip install --upgrade powerline-shell`.

**Comment by b-ryan** [comment]
https://github.com/banga/powerline-shell/commit/655a12a278bb5590b64d531ad3964e58666c903a
